src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -735,15 +735,21 @@ public class Views : RevitDropDownBase
         protected override SelectionState PopulateItemsCore(string currentSelection)
         {
             Items.Clear();
+
             //find all views in the project
-            var fec = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
-            var views = fec.OfClass(typeof(View)).ToElements();
+            //exclude <RevisionSchedule> (revision tables on sheets) from list
+            var views = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(View))
+                .Where(x => !x.Name.Contains('<'))
+                .ToList();
```

Comment:
Yeah by default it returns `Element` collection but also i removed it because I don't think I can call that method after I have used a Linq method so I used the `ToList()` instead.